client/src/plugins/zeebe-plugin/deployment-plugin/DeploymentPluginOverlay.js

```diff
@@ -123,6 +125,27 @@ export default function DeploymentPluginOverlay(props) {
     };
   }, [ connectionChecker, deployment, setConfig, setConnectionCheckResult ]);
 
+  useEffect(() => {
+    const onDeployed = ({ deploymentResult, endpoint, gatewayVersion }) => {
+      triggerAction('emit-event', {
+        type: deploymentResult.success ? 'deployment.done' : 'deployment.error',
+        payload: {
+          deployment: deploymentResult.response,
```

Comment:
This is where the error is passed.

Reply:
See old version of the code: https://github.com/camunda/camunda-modeler/blob/v5.35.0/client/src/plugins/zeebe-plugin/deployment-plugin/DeploymentPlugin.js#L572